plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/shipping/shipping-address.tsx

```diff
@@ -32,15 +35,26 @@ export const ShippingAddress = ( {
 	const prefersCollection = useSelect( ( select ) =>
 		select( CHECKOUT_STORE_KEY ).prefersCollection()
 	);
+	const activeShippingZones = getSetting( 'activeShippingZones' );
+
+	const hasMultipleAndDefaultZone =
+		activeShippingZones.length > 1 &&
+		activeShippingZones.some(
+			( zone: { description: string } ) =>
+				zone.description === 'Everywhere'
+		);
```

Comment:
```suggestion
			( zone: { description: string } ) =>
				zone.description === 'Everywhere' ||
				zone.description === 'Locations outside all other zones'
		);
```

This code should solve the issue I mentioned before.